sequelizemm library added to resource page. (#455)
sequelize/website: docs/other-topics/resources.md

* [umzug](https://github.com/sequelize/umzug) - framework-agnostic migration tool for Node.
* [sequelize-cli](https://github.com/sequelize/cli) - The Sequelize Command Line Interface. Includes umzug-based migrations.
* [sequelize-mig](https://github.com/MRVMV/sequelize-mig) - Sequelize migration generator

> * [sequelizemm](https://github.com/hasinoorit/sequelizemm) CLI tool to generate migration script from models within minute.

Reviewer: ```suggestion
* [sequelizemm](https://github.com/hasinoorit/sequelizemm) - CLI tool to generate a migration script from models
```
